Api/Api.cs

```diff
@@ -443,17 +453,28 @@ private string CreatePathForDownloadedData(Symbol symbol, Resolution resolution,
             string path;
 
             if (resolution == Resolution.Daily || resolution == Resolution.Hour)
-                path = Config.Get("data-folder") + symbol.ID.SecurityType.ToLower() + "/" + symbol.ID.Market.ToLower() +
+                path = _dataFolder + symbol.ID.SecurityType.ToLower() + "/" + symbol.ID.Market.ToLower() +
                        "/" + resolution.ToLower() + "/" + symbol.Value.ToLower() + ".zip";
             else
-                path = Config.Get("data-folder") + symbol.ID.SecurityType.ToLower() + "/" + symbol.ID.Market.ToLower() +
+                path = _dataFolder + symbol.ID.SecurityType.ToLower() + "/" + symbol.ID.Market.ToLower() +
```

Comment:
Path.Combine() is a safe cross platform alternative here; or even better use the:
"LeanData.GenerateRelativeZipFilePath" method in the toolbox LEAN Data Writer,